pypeit/fluxspec.py

```diff
@@ -362,7 +365,7 @@ def generate_sensfunc(self):
 
         self.sens_dict = {}
         try:
-            this_wave = self.std.optimal['WAVE_GRID']
+            this_wave = self.std.boxcar['WAVE_GRID']
         except KeyError:
```

Comment:
I'm not convinced this makes a difference, and would like you to show a plot of both optimal and boxcar showing the differences before changing this.

Reply:
ok, am moving back for now.